views/partials/subscribe_modal/confirm_your_account.hbs

```diff
@@ -1,6 +1,7 @@
 <div id="confirm-your-account" class="modal-content">
   <h3 class="section-headline">Verify Your Subscription</h3>
-  <img src="img/confirm-your-account.svg" />
-  <p>Check your email for a verification link.</p>
+  <img class="confirm-your-account-icon" src="img/confirm-your-account.svg" />
+  <p>Check your email for the verification link sent to <span id="submitted-email" class="medium"></span></p>
```

Comment:
Looks like we are dynamically injecting the users email here. Do we need to sanitize it at all in case of XSS or something?